src/linear_algebra/tensor_product.lean

```diff
@@ -372,6 +396,13 @@ linear_equiv.of_linear (lift $ linear_map.lsmul R M) (mk R R M 1)
   (linear_map.ext $ λ _, by simp)
   (ext $ λ r m, by simp; rw [← tmul_smul, ← smul_tmul, smul_eq_mul, mul_one])
 
+@[simp] theorem lid_tmul (m : M) (r : R) :
+  ((@tensor_product.lid R _ M _ _) : (R ⊗ M → M)) (r ⊗ₜ m) = r • m :=
```

Comment:
Do we want the `@` and all the underscores? Or should some variables be made explicit in another def?

Reply:
Fixed.